deis-dev/manifests/deis-workflow-manager-rc.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    app: deis-workflow-manager
+  template:
+    metadata:
+      labels:
+        app: deis-workflow-manager
+    spec:
+      serviceAccount: deis-workflow-manager
+      containers:
+      - name: deis-workflow-manager
+        image: quay.io/deisci/workflow-manager:v2-beta
```

Comment:
We'll actually want to specify the latest `git-<sha>` in here. So as of right now that'd be `git-941cb10` (from https://quay.io/repository/deisci/workflow-manager?tab=tags). @vdice will help you craft the Jenkins job that will keep that sha up to date without any manual effort.